pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -471,5 +471,5 @@ func (d *Driver) supportsVirtiofs() bool {
 	if err != nil {
 		return false
 	}
-	return strings.HasPrefix(macosVersion, "12.")
+	return strings.HasPrefix(macosVersion, "12.") || strings.HasPrefix(macosVersion, "13.")
```

Comment:
It would be more future proof to use something like https://github.com/Code-Hex/vz/blob/6e20b48bc30d888e13576559826867b8ff4222c0/osversion.go
If we add 13 now, we'll need to add 14 in a year, ...

Reply:
@cfergeau sure, I was just checking what would be the best way to do it for future proof. Will update the PR 👍🏼